src/popup/components/gravatar/gravatar.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Md5 } from "ts-md5";
+import './gravatar.scss';
+
+const Gravatar: React.FC<{ email: string, size?: string | number }> = ({email, size = 30}) => {
+    const emailHash = Md5.hashStr(email || '') as string;
+    const defaultImg = 'https://bitpay.com/img/bp-empty-profile-icon.png';
+    const url = `https://gravatar.com/avatar/${emailHash}.jpg?s=${size}&d=${defaultImg}`;
```

Comment:
Can we do something like `s=${Math.round(size * 2)}` to sharpen the image a bit?